ambry-clustermap/src/main/java/com.github.ambry.clustermap/Datacenter.java

```diff
@@ -38,7 +38,7 @@
 
   private Logger logger = LoggerFactory.getLogger(getClass());
 
-  public Datacenter(HardwareLayout hardwareLayout, JSONObject jsonObject, ClusterMapConfig clusterMapConfig)
+  Datacenter(HardwareLayout hardwareLayout, JSONObject jsonObject, ClusterMapConfig clusterMapConfig)
```

Comment:
Could you add javadoc for this method? It is not very clear what kind information the `JSONObject` would contain. Also, it is a bit strange a DC is instantiated with its key (i.e., its name) hidden in a `JSONObject`.